### PR TITLE
Digital fund preview feature

### PR DESCRIPTION
**Pull request by mattandrews**
Adds a preview flag so test users can complete the full user journey.

### CODE REVIEW

File: middleware/locals.js [hunk 1]
```diff
@@ -169,7 +169,11 @@ module.exports = {
             } else {
                 return false;
             }
-        };
+        }
+
+        /* Configure per-feature queries for use in templates */
+        res.locals.featureUseNewHeader = queryFeature('use-new-header');
+        res.locals.featurePreviewDigitalFund = queryFeature('preview-digital-fund');
```
Comment (davidrapson) [inline middleware/locals.js]: I probably would've put the `appData.isNotProduction` line here so you don't need to repeat it in templates

Comment (mattandrews) [reply middleware/locals.js]: Yeah I wondered about that – possibly there's a situation where we want a feature to be usable even in production?

Comment (davidrapson) [reply middleware/locals.js]: Putting that check here wouldn't prevent that use-case would it? We're just moving the check up.

**Comment by davidrapson** [comment]
This was a much bigger chef than anticipated

**Pull request merged**